tailwindcss/deps.ts

```diff
@@ -0,0 +1,31 @@
+import type { Page } from "lume/core.ts";
+
+import dbin from "dbin/mod.ts";
+
+export async function getTwBinFullPath(
+  version: string,
+  dir: string
+): Promise<string> {
+  return await dbin({
+    pattern: `https://github.com/tailwindlabs/tailwindcss/releases/download/v{version}/tailwindcss-{target}`,
+    version,
+    targets: [
+      { name: "linux-x64", os: "linux", arch: "x86_64" },
+      { name: "linux-arm64", os: "linux", arch: "aarch64" },
+      { name: "macos-x64", os: "darwin", arch: "x86_64" },
+      { name: "macos-arm64", os: "darwin", arch: "aarch64" },
+      { name: "windows-x64.exe", os: "windows", arch: "x86_64" },
+    ],
+    dest: `${dir}/tailwindcss`,
+  });
+}
+
+export function addCssLink2Head(page: Page) {
+  const { document } = page;
+  if (!document) return;
+  const cssLinkEl = document.createElement("link");
+  cssLinkEl.setAttribute("href", "css/main.css");
+  cssLinkEl.setAttribute("rel", "stylesheet");
+  if (!Array.from(document.head.children).includes(cssLinkEl))
```

Comment:
This condition always returns `true` because the `cssLinkEl` element is created here, so it's imposible to be present in the document header. Even if there're another `link` with the same properties, it's a different element.